platform/app-manager/app-manager.js

```diff
@@ -18,71 +21,109 @@ mongoClient.connect(mongoUrl, { useNewUrlParser: true }, function(err, client) {
     db = client.db(appsDb);
 });
 
-// Create logs directory if not present
+// Create logs directory for apps if not present
 fs.ensureDirSync(`${__dirname}/logs`);
 /**
  * This function saves the app info to the database
- * @param {string} appPath application path
+ * @param {string} appName application name
  * @param {string} topic mqtt topic
  * @param {string} pid application pid
- * @returns {string}
```

Comment:
Why is the pid stored in the db?
Why is the appPath and metadataPath not stored in db? (what if there is some failure and we need to restart the app? can we do that without these paths?)

Reply:
1. Why is the pid stored in the db?
If the `platform-manager` is down or `app-manager` is down but the app process still exited, `app-manager` needs to check that before restarting the app.

2. Why is the appPath and metadataPath not stored in db?
~~The path for both app and metadata are the same in executables.~~ I will fix that.

Reply:
_id, appName, appPath, metadataPath, pid